docs/bnb_smart_chain/slashing/slash-events.md

```diff
@@ -0,0 +1,50 @@
+# BSC Slashing Events
+
+Three types of malicious behaviors can lead to slashing on the BSC network.
+
+## Double Sign
+
+Anyone can submit a slash request with evidence of Double Sign. The evidence must adhere to the following rules:
+
+* Two block headers have the same height and the same parent block hash
+* Two block headers are sealed by the same validator
+* Two signatures of these two blocks must not be the same
+* The time of these two blocks must be within the validity of the evidence, which is 24 hours
+
+If the evidence is valid:
+
+1. **10,000BNB** would be slashed from the **self-delegated** BNB of the validator
```

Comment:
The number here is wrong, please refer  to https://docs.google.com/spreadsheets/d/1uoabi2yHZzqH6aMT1NaQ_RrI-J_FlGmkGqZNhYv8UXY/edit#gid=511049805

Reply:
fixed